public/data/food/ingredients.json

```diff
@@ -1,566 +1,839 @@
 [
   {
     "id": "egg",
-    "name": "Oeuf",
-    "category": "animal_product",
+    "name": "Egg, national average, at farm gate {FR} U",
```

Comment:
We actually want to keep the names in French, so they're searchable by French-only speaking people.

Reply:
Thx, looks like an error, `name` should be `default`

Reply:
Hmm, `name` should be `Œuf` and `default` should still be `AGRIBALU000000003105162`.